lib/iterators.js

```diff
@@ -99,29 +99,29 @@ const BlockFace = {
   EAST: 5
 }
 
-// This iterate along a ray starting at (x,y,z) in (dx,dy,dz) direction
+// This iterate along a ray starting at `pos` in `dir` direction
 // It steps exactly 1 block at a time, returning the block coordinates
 // and the face by which the ray entered the block.
 class RaycastIterator {
-  constructor (x, y, z, dx, dy, dz, maxDistance) {
+  constructor (pos, dir, maxDistance) {
```

Comment:
Could this be a breaking change for existing projects that use the old constructor? This class is a public class. Might check the parameter types for backward compatibility.

Reply:
it was undocumented and added recently, I don't think its used anywhere

Reply:
Oh, I thought these iterators were all publicly exported for plugin use. Apparently, they aren't okay then, nevermind. xD